include/libnds_defs.h

```diff
@@ -1,5 +1,5 @@
-#ifndef DEVKITPRO_DEFS__
-#define DEVKITPRO_DEFS__
+#ifndef LIBNDS_DEFS__
```

Comment:
No, the point is to try and use `<nds.h>` instead of redefining symbols...

Reply:
Oh, yeah. I know. LOL
I just fixed the more "immediate" issues with the commit, like the tabs, now I'll look into that instead.

Reply:
The more I look at it, the more I find it weird that the code that this was made for has an ifdef for SYS_GBA, when it directly references NDS only registers...

Reply:
`REG_IME` is also a GBA register, but `REG_SOUNDXCNT` isn't.

Reply:
Yeah, exactly. StopActiveChannel is the function. Which is why it's so weird.
My guess would be some copy-pasted code.